app/lib/linters/spectral.rb

```diff
@@ -0,0 +1,15 @@
+module Linters
+  class Spectral
+    def initialize(file:)
+      @file = file
+    end
+
+    def lint_to_json
+      `spectral lint -f json "#{file.path}"`
+    end
```

Comment:
If this object is only ever going to be used to return the json derived from the input file, I'd suggest adding a class method:
```ruby
def self.lint_to_json(file:)
  new(file).lint_to_json
end
```
Then the controller line 20 becomes:
```ruby
linter_output = Linters::Spectral.lint_to_json(file: ruleset_params[:oas_file])
```

Reply:
Yeah that's a good shout, cheers for that, I'll make that change